### ISSUE TITLE
Problem with env_file (docker compose) overriding

### ISSUE THREAD

**Issue by Gregoire-M** [rancher-sandbox/rancher-desktop#2865]
### Actual Behavior

I have a docker compose project, with several env files. But it seems that only ".env" file is used.

### Steps to Reproduce

Create a project with 3 files as described in [this gist](https://gist.github.com/fabienduhamel/93f408ce426e5155e3647296232840fd).

Then run `docker-compose run --rm -ti alpine env`.

### Result

```
...
APP_ENV=test
...
```

### Expected Behavior

```
...
APP_ENV=dev
...
```

### Additional Information

Docker compose documentation : https://docs.docker.com/compose/compose-file/compose-file-v3/#env_file

This works fine with Docker Desktop.

I'm trying to replace it with Rancher Desktop, but it fails here.

### Rancher Desktop Version

1.5.1

### Rancher Desktop K8s Version

1.24.4

### Which container engine are you using?

moby (docker cli)

### What operating system are you using?

macOS

### Operating System / Build Version

macOS Monterey 12.5.1

### What CPU architecture are you using?

arm64 (Apple Silicon)

**Comment by adamkpickering** [comment]
Thanks for filing an issue!

I was able to reproduce this on Linux (Pop OS 20.04). However, I don't think this is a Rancher Desktop problem. I think that the problem is with the version of `docker-compose` - see this issue: https://github.com/docker/compose/issues/9725.

I was able to fix the problem by updating `docker-compose` to the latest version. I will create a PR to update the version of `docker-compose` RD bundles to latest, but that won't be available until the 1.6.0 release of RD (which we are hoping to have out in a few weeks).

Until then, a workaround is to get the latest version of `docker-compose` from the [`docker-compose` releases page](https://github.com/docker/compose/releases) and put it into `~/.docker/cli-plugins/`. Your issue should then be resolved. Let me know if that doesn't work!
